### PR TITLE
chore: update names for image and readme

### PR DESCRIPTION
**Pull request by Edgeman-SelfID**
updates the name of the images and README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# tbtc-node
+# bitcoin-docker
 
 Builds a bitcoin `testnet4` full node in a docker container
```
Comment (macterra) [inline README.md]: You can remove `testnet4` here, it just build a bitcoin node.

File: .github/workflows/docker-publish.yml [hunk 2]
```diff
@@ -45,9 +45,9 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - dockerfile: Dockerfile.tbtc
+          - dockerfile: Dockerfile.bitcoin-core
```
Comment (macterra) [inline .github/workflows/docker-publish.yml]: The file can be named just `Dockerfile`

**Review by macterra — CHANGES_REQUESTED** [review]
Comments inline

File: README.md (hunk 1 above)
Comment (Edgeman-SelfID) [reply README.md]: fixed

File: .github/workflows/docker-publish.yml (hunk 2 above)
Comment (Edgeman-SelfID) [reply .github/workflows/docker-publish.yml]: fixed

**Review by macterra — APPROVED** [review]

**Pull request merged**